source/_imports/components/bootstrap.pagination.css

```diff
@@ -0,0 +1,29 @@
+/*
+Pagination (Bootstrap)
+
+Style Bootstrap pagination. See:
+
+- [Bootstrap Pagination](https://getbootstrap.com/docs/4.0/components/pagination/)
+
+Styleguide Components.Bootstrap.Pagination
+*/
+@import url("@tacc/core-styles/source/_imports/components/c-page.css");
```

Comment:
**Hint:** The [imported stylesheet](https://github.com/TACC/Core-Styles/blob/main/source/_imports/components/c-page.css) was added in https://github.com/TACC/Core-Styles/pull/5.